central/search/service/service_impl_test.go

```diff
@@ -110,25 +112,46 @@ type SearchOperationsTestSuite struct {
 	mockCtrl *gomock.Controller
 	rocksDB  *rocksdb.RocksDB
 	boltDB   *bolt.DB
+	ctx      context.Context
+	db       *pgxpool.Pool
+	gormDB   *gorm.DB
 }
 
 func (s *SearchOperationsTestSuite) SetupSuite() {
 	s.envIsolator = envisolator.NewEnvIsolator(s.T())
 }
 
 func (s *SearchOperationsTestSuite) SetupTest() {
-	s.envIsolator.Setenv(features.NewPolicyCategories.EnvVar(), "true")
+	s.envIsolator.Setenv(features.NewPolicyCategories.EnvVar(), "false")
+	s.envIsolator.Setenv(features.PostgresDatastore.EnvVar(), "false")
+
 	s.mockCtrl = gomock.NewController(s.T())
 	s.rocksDB = rocksdbtest.RocksDBForT(s.T())
 	var err error
 	s.boltDB, err = bolthelper.NewTemp(s.T().Name() + "-bolt.db")
 	s.NoError(err)
+
+	s.ctx = context.Background()
+	if features.PostgresDatastore.Enabled() {
```

Comment:
This is never true because of above `SetEnv`

Reply:
The search service never worked for postgres btw. The way the postgres instance was created for this test would never work with feature flag turned on.

Reply:
And "This is never true because of above SetEnv" has always been true :)